sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_models.py

```diff
@@ -147,8 +147,6 @@ def __init__(self, key, enabled, filters=None, **kwargs):
         self._enabled = enabled
         super(FeatureFlagConfigurationSetting, self).__init__(**kwargs)
         self.key = key
-        if not key.startswith(self.key_prefix):
```

Comment:
Do we want to check if key.startswith(self.key_prefix)?

Reply:
Do we still want to do this? I removed it to align with java and javascript/typescript.

Reply:
if not key.startswith(self.key_prefix):
    raise ValueError. :)

Reply:
Added!

Reply:
There was some confusion in our discussion, the standard is to prefix no matter what. That is what Java and C# do, I will add the same to Python